Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -110,12 +110,22 @@ extension SettingsDictionary {
 
     /// Sets `"OTHER_SWIFT_FLAGS"` to `flags`
     public func otherSwiftFlags(_ flags: String...) -> SettingsDictionary {
```

Comment:
If we favore the array one (as we don't have this for other settings) we should deprecate these onse

Reply:
Added deprecation notice